.circleci/config.yml

```diff
@@ -1939,22 +1957,14 @@ jobs:
           checkout-and-assume-cache: true
           attach: false
 
-  osx-testing-gn-check:
+  osx-testing-x64-gn-check:
     <<: *machine-mac
     environment:
       <<: *env-machine-mac
       <<: *env-testing-build
     <<: *steps-electron-gn-check
 
-  osx-chromedriver:
-    <<: *machine-mac
-    environment:
-      <<: *env-machine-mac
-      <<: *env-release-build
-      <<: *env-send-slack-notifications
-    <<: *steps-chromedriver-build
```

Comment:
Was it intentional to remove the chromedriver build?

Reply:
This build was unused and completely unreferenced